tsl/test/expected/continuous_aggs_errors.out

```diff
@@ -473,30 +473,37 @@ select time_bucket(100, timec), min(location), sum(temperature),sum(humidity)
 from conditions
 group by time_bucket(100, timec);
 ERROR:  parameter timescaledb.ignore_invalidation_older_than must be an integer for hypertables with integer time values
-\set ON_ERROR_STOP 1
 create materialized view mat_with_test( timec, minl, sumt , sumh)
 WITH ( timescaledb.continuous, timescaledb.refresh_lag = '2147483647')
 as
 select time_bucket(100, timec), min(location), sum(temperature),sum(humidity)
 from conditions
 group by time_bucket(100, timec);
--- Should print a useful error message, but not fail.
-CREATE MATERIALIZED VIEW IF NOT EXISTS mat_with_test
-WITH (timescaledb.continuous) AS
-SELECT time_bucket(100, timec), min(location), sum(temperature),sum(humidity)
-  FROM conditions
-GROUP BY time_bucket(100, timec);
-NOTICE:  continuous aggregate "mat_with_test" already exists, skipping
+ERROR:  invalid integer_now function
```

Comment:
This statement now suddenly starts to fail when it succeeded before. Is this intentional?

Reply:
yes, current error checking on alter type appears to be inadequate. This si the correct behavior.